openhtf/plugs/usb/__init__.py

```diff
@@ -36,9 +38,27 @@ def MyPhase(test, adb):
 conf.Declare('usb_server', 'USB Server IP/Hostname')
 conf.Declare('usb_server_port', 'USB Server Port',
              default_value=10000)
-
 conf.Declare('libusb_rsa_key', 'A private key file for use by libusb auth.')
+conf.Declare('remote_usb', 'Ethersyncy or other',default_value=None)
+
+_LOG = logging.getLogger(__name__)
 
+class GeneralUsbAttributeError(Exception):
+  """Indicates the usb class/subclass/protocol info was invalid."""
+
+def _get_usb_serial(port_addr):
```

Comment:
We should strongly consider wrapping the esuit64 tool in an EtherSync object abstraction. This would make EtherSync interactions more declarative, would be usable as a straight 'EtherSync' plug if we need to do other things besides just get the serial for making a handle (like switch the USB ports on and off), and so that when Cambrionix comes out with a native python API we can easily switch over to that under the hood.

This would also give us a clean(er) place to do error checking. For example, if the esuit64 tool doesn't exist in the path, we should be able to detect that and give a useful error message.